backend/src/main/resources/templates/uploadGallery.html

```
@@ -0,0 +1,190 @@
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<template id="name-input-template">
+    <div class="gallery-upload-card">
+        <div class="preview-container">
+            <img class="image-preview" src="" alt="preview">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)